monkey/infection_monkey/exploit/zerologon_utils/remote_shell.py

```diff
@@ -71,6 +71,7 @@ def __init__(self, share, win32Process, smbConnection, outputFilename, secrets_d
         self.__secrets_dir = secrets_dir
 
         # We don't wanna deal with timeouts from now on.
+        # TODO are we sure we don't need timeout anymore?
```

Comment:
@VakarisZ do we? Maybe it's worth putting some kind of timeout where the remote shell is used in `zerologon.py` (in `save_HKLM_keys_locally()`).

Reply:
Anything that can block indefinitely needs a timeout, which we can resolve in #1611.